main.go

```diff
@@ -36,16 +36,16 @@ import (
 )
 
 const (
-	secretAnnotKey = "cert-manager.io/secret-transform"
-	tlsPEMDataKey  = "tls.pem"
+	secretAnnotKey = "cert-manager-secret-transform"
+	tlsDERDataKey  = "tls.der"
```

Comment:
Was this change required due to some limitation? Just wondering :grin:

Reply:
> Very nice! Do you think we could have both support for PEM and DER formats? I was thinking about using the extension (pem, der, p12, or pks) in order to choose which encoding should be used.

I currently [working to add cert-manager additional key format support](https://github.com/jetstack/cert-manager/pull/4598).
The idea is to add both `PEM` and `DER` format support.